File: solutions/dart/raindrops/1/lib/raindrops.dart
// Offered in reverence to Shree DR.MDD 🌹

class Raindrops {
  String convert(int val) {
    var sound = StringBuffer();
    if (val % 3 == 0) sound.write('Pling');
    if (val % 5 == 0) sound.write('Plang');
    if (val % 7 == 0) sound.write('Plong');
    if (sound.isEmpty) sound.write(val);
    return sound.toString();
  }
}

File: solutions/lua/raindrops/1/raindrops.lua
return function(n)
    local drops = string.format(
        "%s%s%s",
        n % 3 == 0 and "Pling" or "",
        n % 5 == 0 and "Plang" or "",
        n % 7 == 0 and "Plong" or ""
    )
    return drops == "" and tostring(n) or drops
end